lib/Rex/Commands/Run.pm

```diff
@@ -183,6 +183,26 @@ sub run {
   $option->{auto_die} = Rex::Config->get_exec_autodie()
     if !exists $option->{auto_die};
 
+  # We only process cmd is an explicit path that means
+  # that starts with . .. <letter>: and / or \
+  # and aren't scaped spaces or quotes
+  {
```

Comment:
Also, do we need to always run this logic, or only when there's a space in the command?

Reply:
The code block is here oniy to encapsulate the variables but as we need only to run it if there are spaces it could be better write as if ($cmd =~ /\ /msx ) {